src/Canvas/Pixel.elm

```diff
@@ -0,0 +1,227 @@
+module Canvas.Pixel
+    exposing
+        ( put
+        , putMany
+        , line
+        , rectangle
+        , bezier
+        )
+
+import Canvas exposing (Canvas, Size, Point, DrawOp(..))
+import Color exposing (Color)
+import Array exposing (Array)
+
+
+put : Color -> Point -> DrawOp
```

Comment:
Oh, point is fine on this. My gripe is that `putHelp` and `put` are basically the same method other than one takes a tuple and the other breaks the args out.